src/index.ts

```diff
@@ -12,5 +12,3 @@ export * from './localnet'
 export * from './network-client'
 export * from './transaction'
 export * from './transfer'
-export * from './types/algorand-client'
```

Comment:
While these classes are in beta it makes sense to not expose them from index so you have to explicitly import them

Reply:
As discussed on Slack, will add this back to index but preserve the beta tag

Reply:
Exported in [6cf626b](https://github.com/algorandfoundation/algokit-utils-ts/pull/261/commits/6cf626b83c2c5565a7f9bbfb22421b577f2ee67b)